### DISCUSSION TITLE
archery 支持slack webhook通知吗？

### DISCUSSION THREAD

**Discussion by yangguocang** [hhyo/Archery#2823] [Q&A]
archery 支持slack webhook通知吗？  如果支持该怎么配置呀？

**Comment by LeoQuote** [comment]
有两个思路：
1. archery 支持general webhook，你可以自己实现handler，把事件转发到slack，详情看代码 https://github.com/hhyo/Archery/blob/073560e91e95dec68084445d6f6abff0faaedd4c/sql/notify.py#L99
2. https://github.com/hhyo/Archery/wiki/notifier 自己实现一个notifier

**Reply by yangguocang** [reply]
有没有一个详细的示例呀  每个工单或者查询都能发送到 slack到webhook url中

**Comment by EddyLhq** [comment]
同问
